.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-prisma/migrations
\ No newline at end of file
+prisma/migrations
```

Comment:
prisma/migrations should be a part of github, please remove it from `.gitignore`